src/Umbraco.Web.UI.Client/src/views/webhooks/overlays/details.html

```diff
@@ -1,9 +1,9 @@
-﻿<div class="headless-webhook-log-entry-details-overlay" style="margin-top: 20px;">
+<div class="headless-webhook-log-entry-details-overlay" style="margin-top: 20px;">
   <div class="control-group">
     <div class="flex items-center">
       <div class="flx-g0 flx-s0" style="flex-basis: 40px;">
         <umb-checkmark checked="true" size="m" style="cursor: default"></umb-checkmark>
-        <i class="icon-wrong umb-checkmark umb-checkmark--m" style="cursor: default;" ng-if="model.webhookLogEntry.response.isSuccess === false"></i>
+        <umb-icon icon="icon-wrong" class="umb-checkmark umb-checkmark--m" style="cursor: default;" ng-if="model.webhookLogEntry.response.isSuccess === false"></umb-icon>
```

Comment:
I didn't have any logs, but we should use `<umb-icon>` instead of the `<i>` element :)